website/source/api/config.html.md

```diff
@@ -0,0 +1,246 @@
+---
+layout: api
+page_title: Config - HTTP API
+sidebar_current: api-config
+description: |-
+  The /config endpoints are for managing centralized configuration
+  in Consul.
+---
+
+# Config HTTP Endpoint
+
+The `/config` endpoints create, update, delete and query central configuration
+entries registered with Consul.
+
+## Apply Configuration
+
+This endpoint creates or updates the given config entry.
+
+| Method | Path                         | Produces                   |
+| ------ | ---------------------------- | -------------------------- |
+| `PUT`  | `/config`                    | `application/json`         |
+
+The table below shows this endpoint's support for
+[blocking queries](/api/index.html#blocking-queries),
+[consistency modes](/api/index.html#consistency-modes),
+[agent caching](/api/index.html#agent-caching), and
+[required ACLs](/api/index.html#acls).
+
+| Blocking Queries | Consistency Modes | Agent Caching | ACL Required  |
+| ---------------- | ----------------- | ------------- | ------------- |
+| `NO`             | `none`            | `none`        | `service:write`<br>`operator:write`<sup>1</sup> |
+
+<sup>1</sup> The ACL required depends on the Kind of the config entry being updated:
```

Comment:
```suggestion
<sup>1</sup> The ACL required depends on the config entry kind being updated:
```

Reply:
@kaitlincarter-hc The "Kind" here is a field that is part of the config entry structure. 

A typical service config would look like this (proxy configs still have a Kind/Name):

```json
{
   "Kind": "service-defaults",
   "Name": "web",
   "Protocol": "http",
   "Connect": {
      "SidecarProxy": true
   }
}
```

Would you still advise this change given that?